StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/USBankAccount/IncentiveOwner.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  IncentiveOwner.swift
+//  StripePaymentSheet
+//
+//  Created by Till Hellmund on 1/14/25.
+//
+
+import Foundation
+
+protocol IncentiveOwner {
```

Comment:
Ideas for better names are welcome!

Reply:
Yep, will do that. We can create a protocol if this expands to other payment methods.